src/ExNavigationTypeDefinition.js.flow

```diff
@@ -40,6 +40,7 @@ export type ExNavigationBarConfig = {
   titleStyle?: Object | number,
   tintColor?: string,
   backgroundColor?: string | Function,
+  backgroundImage?: string | Function,
```

Comment:
@cem2ran 
[https://github.com/exponentjs/ex-navigation/pull/185](url)
Check this out what you think about it!!